modules/docs/mdx/8.0-UPGRADE-GUIDE.mdx

```diff
@@ -78,7 +78,24 @@ above will no longer run in IE11 and will fail to bootstrap with an error and/or
 
 ### React 18 Upgrade
 
-> Coming Soon (https://github.com/Workday/canvas-kit/discussions/1545)
+We've upgraded Canvas Kit to [React 18](https://reactjs.org/blog/2022/03/29/react-v18.html). React
+18 introduces several new features, but the most impactful change is the new
+[concurrent render API](https://reactjs.org/blog/2022/03/29/react-v18.html#what-is-concurrent-react).
+The new `createRoot` method replaces `render` and gives you access to the improvements in React 18
+including the concurrent features. Without it, React behaves as it did in version 17.
+
+Canvas Kit doesn’t use the render API directly in our packages but Storybook uses it to render
+Canvas Kit components. Because of this, we upgraded Storybook dependencies to support React 18.
+
+This change will not impact product teams that are already on v7 and are looking to upgrade to v8 as
```

Comment:
This is looking really good especially with the latest change. I recommend moving this section up to be directly after the first sentence that starts and ends on line 81. 

The content after that sentence would start a new paragraph.

I recommend this to make sure that there is little or no room for confusion around impact to users. Users will be looking for what this means to them as soon as they hear that we're upgraded to React 18.

Reply:
Amending the above suggestion based on conversation with @RayRedGoose:

```
### React 18 Upgrade

We've upgraded Canvas Kit to [React 18](https://reactjs.org/blog/2022/03/29/react-v18.html). This
change will not impact users who are already on v7 and are looking to upgrade to v8 as v8 will
support the same React versions as v7 (React 16.14 and 17.X) while still giving them the ability to
upgrade to React 18 as needed. Although we didn't use any new features from React 18 in v8, _future_
updates to Canvas Kit may require upgrading to React 18 to support them.

React 18 introduces several new features...
```